src/electron/views/renderer-initializer.ts

```diff
@@ -351,6 +362,23 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then(
 
         const documentManipulator = new DocumentManipulator(document);
 
+        const reportHtmlGenerator = new ReportHtmlGenerator(
+            UnifiedReportSectionFactory,
+            new ReactStaticRenderer(),
+            { extensionVersion: '1.1', browserSpec: "doesn't exist", axeCoreVersion: '0.16' },
```

Comment:
I guess this may have conflicts with the other PR that refactor how we pass environment info, right? Just a heads up in case you forgot.